src/matrix/__tests__/matrixCheckRanges.test.ts

```diff
@@ -0,0 +1,42 @@
+import { matrixCheckRanges } from '../matrixCheckRanges';
+
+describe('matrixCheckRanges', () => {
```

Comment:
We would like in the future to get rid of the `describe` when there is only one.
So please remove the line `describe` and replace all the `it` by `test`